### ISSUE TITLE
Start using individual keys for all new prod certs

### ISSUE THREAD

**Issue by lgarron** [chromium/badssl.com#279]
Using a single key is convenient, but multiple keys aren't that hard to handle, and it avoids putting all eggs in one basket in case something needs to be revoked (e.g. because Google revokes by SPKI).

**Comment by ByJamesBurton** [comment]
You really need a piece of software which you can install on the server to monitor the certificates and keys.

**Comment by lgarron** [comment]
> when you need to renew the certificate

That's a completely orthogonal issue.
(Using the same key for every cert makes renewal *easier*, though.)

**Comment by ByJamesBurton** [comment]
Yes you right and its a different issue, I meant using certificate management platform to monitor your certificates and key better without a lot of difficulty.
